src/isomorphic/ReactDebugTool.js

```diff
@@ -48,6 +48,12 @@ var ReactDebugTool = {
       }
     }
   },
+  onBeginProcessingChildContext(instance, boolean) {
```

Comment:
`boolean` is a type.  A good variable name would be something like `isPprocessingChildContext`.  But why does this variable exist at all?  Isn't it always true in this function and always false in the other?